docs/features/datastores/2_Using Datastores/2_creating-datastores.md

```diff
@@ -50,12 +64,66 @@ This method can work from:
+
+Any public [AWS S3](https://aws.amazon.com/s3/) bucket can be used to create datastores on
+the grid public cloud or a BYOC cluster using either the grid CLI or UI. 
 
-At this time we are only supporting usage of private S3 buckets as Datastores for BYOC users, who have connected Grid to a custom AWS cluster. You can grant Grid access to your desired buckets by following the official aws [documentation](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/).
+### Using the UI
 
-As a convenience, below we provided a bucket policy that grants Grid access to all the contents of your specified bucket. It assumes that you modified the tfvars role_arn field. If you have not then you can use `<aws-account-id-associated-with-byoc>:root` instead. You can follow this official aws [documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html) to get your account id.
+**_TODO: CAPTURE SCREEN RECORDING SHOWING STEPS OF CREATING DATASTORE FROM S3 VIA UI_**
+
+### Using the CLI
+
+In order to use the CLI to create a datastore from an S3 bucket, we simply need to pass a
+S3 URL in the form `s3://<bucket-name>/<any-desired-subpaths>/` to the 
+[`grid datastore create`](../../../cli.md#create) command. 
+
+For example, to create a datastore from the [`ryft-public-sample-data/esRedditJson`
+bucket](https://s3.console.aws.amazon.com/s3/buckets/ryft-public-sample-data?region=us-east-1&prefix=esRedditJson/&showversions=false)
+we simply execute:
+
+```bash
+grid datastore create s3://ryft-public-sample-data/esRedditJson/
 ```
+
+Which will copy the files from the source bucket into the managed Grid Datastore storage
+system. 
+
+:::tip
+
+In the above example, you'll see that we omitted the `--name` option in the CLI command.
+When the `--name` option is omitted, the datastore name is assigned the name of the last
+"directory" making up the source path. So in the case above, the datastore would be named
+`"esredditjson"` (the name is converted to all lowercase ASCII non-space characters). 
+
+If we we wanted to use a different name, we can override the implicit naming by passing
+the `--name` option / value parameter explicitly. As an example, if we wanted to create a
+datastore from this bucket named `"lightning-train-data"` we could execute:
+
+```bash
+grid datastore create s3://ryft-public-sample-data/esRedditJson/ --name lightning-train-data
+```
+
+:::
+
+## Create from a Private S3 Bucket
+
+At this time we are only supporting usage of private S3 buckets as Datastores for BYOC
+users, who have connected Grid to a custom AWS cluster. You can grant Grid access to your
+desired buckets by following the official aws
+[documentation](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/).
+
+As a convenience, below we provided a bucket policy that grants Grid access to all the
+contents of your specified bucket. It assumes that you modified the tfvars role_arn field.
```

Comment:
@rlizzo can you add some context about what tfvars is? and why would they have modified it and why can they use that alternative. pretty much, why is it important to them?